### ISSUE TITLE
update_checkstyle_thresholds script failing due to authentication failure

### ISSUE THREAD

**Issue by ssoloff** [triplea-game/triplea#1700]
I first noticed this in build [4220](https://travis-ci.org/triplea-game/triplea/builds/231043149):

```
$ chmod +x ./.travis/update_checkstyle_thresholds && ./.travis/update_checkstyle_thresholds
Checking if Checkstyle thresholds should be updated for this build...
Updating Checkstyle thresholds...
Checking for changes to 'gradle.properties'...
  ...changes detected.
Pushing 'gradle.properties' changes to '[secure]-game/[secure]:master'...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   454  100    93  100   361   3099  12032 --:--:-- --:--:-- --:--:-- 16409
{
  "message": "Bad credentials",
  "documentation_url": "https://developer.github.com/v3"
}
```

This script last ran successfully in build [4186](https://travis-ci.org/triplea-game/triplea/builds/229826092) three days ago (2017-05-08T03:47:38Z).

I believe the authentication failure is due to the recently-reported [Travis security advisory](https://blog.travis-ci.com/2017-05-08-security-advisory).  It is **possible** that the triplea-game token was unintentionally disclosed in an archived log, and the Travis team discovered it during their audit and asked GitHub to revoke the token.

@ron-murhammer, @DanVanAtta Please determine if the GitHub token assigned to the `GITHUB_PERSONAL_ACCESS_TOKEN_FOR_TRAVIS` Travis environment variable has been revoked.  If it has not, I'll investigate further.

**Comment by ssoloff** [comment]
Just noting that both [`update_checkstyle_thresholds`](https://travis-ci.org/triplea-game/triplea/builds/235079237#L5558) and [`push_tag`](https://travis-ci.org/triplea-game/triplea/builds/235079237#L5743) are still failing due to authentication failures as of build 4324.

**Comment by ssoloff** [comment]
:tada: Looks like `update_checkstyle_thresholds` is working again, at least as of build [4579](https://travis-ci.org/triplea-game/triplea/builds/239804088).

@DanVanAtta It appears you were testing the fix with #1816.  Can you explain what you did to fix it?

**Comment by DanVanAtta** [comment]
Certainly. I logged in to the bot account and:
- cleaned out all the map keys (There were quite a lot, 2 per map, over 200+)
- with that reduced, it was easy to see we only had 2 keys configured. I added the missing 'push_tags' key and updated the travis website to have these. I started some documentation here on the builder  bot and maintenance: https://github.com/triplea-game/triplea-game.github.io/pull/218
